src/validation/busValidation.js

```diff
@@ -22,13 +14,26 @@ exports.busValidate = (req, res, next) => {
             "string.empty": res.__('company should not be empty')
         }),
 
+        type: Joi.string().required().messages({
+            "any.required": res.__('type should be required'),
+            "string.empty": res.__('type should not be empty')
+        }),
+
+        location: Joi.string().required().messages({
+            "any.required": res.__('location should be required'),
+            "string.empty": res.__('location should not be empty')
+        }),
+
         seats: Joi.number().integer().min(18).required().messages({
             "any.required": res.__('seats should be required'),
-        }, displayMessage(res)),
+            "number.base": res.__('seats should be a number'),
+            "number.integer": res.__('seats should be integer number'),
+            "number.min": res.__('seats should be greater than or equal to 18')
+        }),
 
-        status: Joi.string().valid('active', 'inactive').required().messages({
+        status: Joi.string().valid('start', 'moving', 'stop').required().messages({
             "any.required": res.__('status should be required'),
-            "any.only": res.__("status should be [active, inactive]")
+            "any.only": res.__("status should be ['start', 'moving', 'stop']")
```

Comment:
what can be the difference between start and moving statuses?

According to trello we must have statuses: at rest, moving or stuck in traffic

Reply:
solved.